modules/apps/cookies/cookies-banner-web/src/main/resources/META-INF/resources/js/CookiesUtil.js

```diff
@@ -47,4 +48,6 @@ export function setCookie(name, value) {
 
 export function setUserConfigCookie() {
 	setCookie(userConfigCookieName, 'true');
+
+	getOpener().Liferay.fire('cookieBannerSetCookie');
```

Comment:
One last thought, can `getOpener()` ever return null or undefined? If so, let's add a check for that.

If it can't ever be, feel free to forward

Reply:
Sure thing. I'll do it.